docs/examples/coreos-kops-tests-multimaster.md

```diff
@@ -0,0 +1,310 @@
+||  RootDeviceName     |  /dev/xvda                                     ||
+||  RootDeviceType     |  ebs                                           ||
+||  SriovNetSupport    |  simple                                        ||
+||  State              |  available                                     ||
+||  VirtualizationType |  hvm                                           ||
+|+---------------------+------------------------------------------------+|
+|||                         BlockDeviceMappings                        |||
+||+-----------------------------------+--------------------------------+||
+|||  DeviceName                       |  /dev/xvda                     |||
+|||  VirtualName                      |                                |||
+||+-----------------------------------+--------------------------------+||
+||||                                Ebs                               ||||
+|||+------------------------------+-----------------------------------+|||
+||||  DeleteOnTermination         |  True                             ||||
+||||  Encrypted                   |  False                            ||||
+||||  SnapshotId                  |  snap-00d2949d7084cd408           ||||
+||||  VolumeSize                  |  8                                ||||
+||||  VolumeType                  |  standard                         ||||
+|||+------------------------------+-----------------------------------+|||
+|||                         BlockDeviceMappings                        |||
+||+----------------------------------+---------------------------------+||
+|||  DeviceName                      |  /dev/xvdb                      |||
+|||  VirtualName                     |  ephemeral0                     |||
+||+----------------------------------+---------------------------------+||
+```
+
+Also, you can obtaing the owner/name using the command (that is what we really need for KOPS):
+
+```bash
+aws ec2 describe-images --region=us-east-1 --owner=595879546273 \
+    --filters "Name=virtualization-type,Values=hvm" "Name=name,Values=CoreOS-stable*" \
+    --query 'sort_by(Images,&CreationDate)[-1].{id:ImageLocation}' \
+	--output table
+
+	
+---------------------------------------------------
+|                 DescribeImages                  |
++----+--------------------------------------------+
+|  id|  595879546273/CoreOS-stable-1409.8.0-hvm   |
++----+--------------------------------------------+
+```
+
+This "image id" (595879546273/CoreOS-stable-1409.8.0-hvm) is the one we'll need later in order to change KOPS default (kope.io/k8s-1.6-debian-jessie-amd64-hvm-ebs-2017-05-02) for the one running CoreOS.
+
+
+## KOPS CLUSTER CREATION AND MODIFICATION:
+
+Let's first create our cluster:
+
+```bash
+kops create cluster \
+--master-zones=us-east-1a,us-east-1b,us-east-1c \
+--zones=us-east-1a,us-east-1b,us-east-1c \
+--node-count=2 \
+${NAME}
+```
+
+NOTE: Remember ${NAME} was exported with our cluster name: coreosbasedkopscluster.k8s.local
```

Comment:
I think it would be clearer if you set NAME just before you do `kops create cluster`, because then we differentiate between the things that are one-off and the things that are per-cluster.